verilog/formatting/tree_unwrapper.cc

```diff
@@ -701,6 +711,74 @@ void TreeUnwrapper::Visit(const verible::SyntaxTreeNode& node) {
       }
       break;
     }
+    case NodeEnum::kGenerateIfClause:
+    case NodeEnum::kGenerateIfBody: {
+      const verible::SyntaxTreeNode* subnode;
+
+      if (tag == NodeEnum::kGenerateIfClause)
+        subnode = &GetSubtreeAsNode(node, NodeEnum::kGenerateIfClause, 1);
```

Comment:
In general, I'd like to start moving these subtree accesses into CST helper functions (one-liners) to isolate some of the structural details.  Perhaps this could be done as a follow-up clean-up.